modules/videoio/src/cap_ios_video_camera.mm

```diff
@@ -118,6 +122,10 @@ - (void)start;
 
 - (void)stop;
 {
+    if (self.running == NO) {
```

Comment:
Looks like self.running could became equal to NO if one calls to [CvVideoCamera pause]. So in such case proper resource deallocation would be skipped.